quic/client/test/QuicClientTransportTest.cpp

```diff
@@ -1080,7 +1081,9 @@ INSTANTIATE_TEST_CASE_P(
 // from the server.
 class FakeOneRttHandshakeLayer : public ClientHandshake {
  public:
-  explicit FakeOneRttHandshakeLayer(QuicClientConnectionState* conn)
+  explicit FakeOneRttHandshakeLayer(
+      QuicClientConnectionState* conn,
+      std::shared_ptr<FizzClientQuicHandshakeContext> fizzContext)
```

Comment:
This has an unused param warning which ends up being a compiler error internally

Reply:
Ok will fix. I plan to use it down the road, but I can introduce it then rather than now.